src/components/victory-bar/victory-bar.js

```diff
@@ -408,15 +408,17 @@ export default class VictoryBar extends React.Component {
         {}, dataProps, {events: Events.getPartialEvents(dataEvents, key, dataProps)}
       ));
 
-      const labelProps = defaults(
-        {index, key: `${role}-label-${key}`},
-        this.getEventState(key, "labels"),
-        this.getSharedEventState(key, "labels"),
-        labelComponent.props,
-        this.baseProps[key].labels
-      );
+      const shouldGetLabelProps = (this.baseProps[key].labels && this.baseProps[key].labels.text)
```

Comment:
`&& this.baseProps[key].labels.text` is unnecessary because `baseProps[key].labels` only exists if there was text correct?